src/epcpm/parameterstointerface.py

```diff
@@ -375,7 +375,10 @@ def gen(self):
             variable_or_getter_setter = [
                 f".getter = {getter_function},",
             ]
-
+        
+        if parameter.getter_index is not None:
+            variable_or_getter_setter.append(f".getter_index = {parameter.getter_index},")
```

Comment:
Dont need a setter if its read_only

Reply:
The setter gets the value that is defined in the json. Do you want me to force setter to NULL if the index attribute is present?

Reply:
https://github.com/altendky/stlib/pull/24#discussion_r660625940

Reply:
since the read_only getter index has a value, we dont even have to have a setter = null since it takes up extra memory.  here is an example from my Read Only Constants.

![image](https://user-images.githubusercontent.com/80793882/123825240-9bd36480-d8b3-11eb-8115-b81361f40708.png)

Reply:
Alright, thanks for clarifying. The setter has now been removed